src/pal/inc/pal_mstypes.h

```diff
@@ -218,7 +218,11 @@ typedef __int32 int32_t;
 typedef unsigned __int32 uint32_t;
 typedef __int16 int16_t;
 typedef unsigned __int16 uint16_t;
+
+#if !defined(__ANDROID__)
+// Android already defines these types
 typedef __int8 int8_t;
```

Comment:
There's a couple of locations where they are declared, such as `stdint`, but it appears this is no longer causing any issues when cross-compiling on Android. So I'm going to drop this commit from the PR.